src/core/border_router/routing_manager.cpp

```diff
@@ -1191,6 +1191,7 @@ void RoutingManager::DiscoveredPrefixTable::ProcessRouterAdvertMessage(const Ip6
             ExitNow();
         }
 
+        router->Clear();
         router->mAddress = aSrcAddress;
         router->mEntries.Clear();
```

Comment:
Is `router->mEntries.Clear()` necessary after `router->Clear()`?

Reply:
Good point. We can remove it now that we do `router->Clear()`. Will change in next push.